### ISSUE TITLE
Intel Arc Support

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/Northstar#458]
**TL;DR:** Northstar should work out-of-the-box on Intel Arc. This thread is primarily to collect information on when it might not work optimally and solutions on how to remedy such cases.

With that said...

I recently got my hands on an Intel A770 to replace my Nvidia RTX 2070 (it's barely an upgrade I know, I just wanted AV1 ._.).

Obviously I played around with it on Titanfall2 and Northstar. The following is some notes of stuff I came across:

In general performance is worse than the RTX2070, probably due to ~~DX9~~ DX11 -> 12/Vulkan translation but that could be improved with drivers still.

Both Titanfall2 and Northstar seems to cause longer CPU spikes sometimes. Not sure what the cause of that is. Shader compilation?

I'm running OBS with Replay Buffer at the same time and closing OBS would sometimes stop CPU spike. Not sure if coincidence of correlation. (Note that I checked that CPU isn't software video encoding, I'm using hardware encoder).

[More to add here]

**Comment by F1F7Y** [comment]
Rare Gecko L

**Comment by GeckoEidechse** [comment]
Been running the Intel Arc for quite a while now and basically all issues I have encountered with it on Windows (which is basically just occasional micro stutter) also happen on other games and are not Northstar/Titanfall2 specific.

More importantly I didn't have a single crash that could be attributed to the A770 in my system.

As such I'm closing this as "working". If anyone is running an Intel Arc and has issues with Northstar specifically, either make a new issue or leave a comment under this one and I will re-open.
